### PR TITLE
[bitnami/rabbitmq-cluster-operator] Add-support-for-metrics-endpoint 

### PR DESCRIPTION
**Pull request by basit9958**
### Description of the change
[fixes#19592](https://github.com/bitnami/charts/issues/19592)

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

*PR sponsored by [Obmondo.com](https://obmondo.com)*

### CODE REVIEW

**Comment by carrodher** [comment]
Thank you for initiating this pull request. We appreciate your effort. Just a friendly reminder that it's important to sign your commits. Adding your signature certifies that you either authored the patch or have the necessary rights to contribute the changes. You can find detailed information on how to do this in the [“Sign your work” section of our contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

Feel free to reach out if you have any questions or need assistance with the signing process.

File: bitnami/rabbitmq-cluster-operator/templates/cluster-operator/servicemonitor.yaml [hunk 1]
```diff
@@ -47,4 +47,11 @@ spec:
       {{- if .Values.clusterOperator.metrics.serviceMonitor.relabelings }}
       relabelings: {{ toYaml .Values.clusterOperator.metrics.serviceMonitor.relabelings | nindent 8 }}
       {{- end }}
+    - port: metrics
+    - interval: {{ .Values.clusterOperator.metrics.serviceMonitor.interval }}
+      params:
+      family: {{- range $family := .Values.clusterOperator.metrics.serviceMonitor.params.family}}
```
Comment (jouve) [inline bitnami/rabbitmq-cluster-operator/templates/cluster-operator/servicemonitor.yaml:53]: you can dump a list with toYaml
```suggestion
      family: {{- .Values.clusterOperator.metrics.serviceMonitor.params.family | toYaml | nindent 8 }}
```

**Comment by basit9958** [comment]
@javsalgar @jouve please review this PR

**Comment by Mauraza** [comment]
Hi @basit9958,

Could you check how to appear in [rabbitmq](https://github.com/bitnami/charts/blob/6d3450c901ee3c19a6422578cd755e6d37399670/bitnami/rabbitmq/templates/servicemonitor.yaml#L26-L28)  and do the same here?

**Comment by basit9958** [comment]
cc @Mauraza @carrodher 
made changes please check

**Review by Mauraza — APPROVED** [review]
Hi @basit9958,

Thanks for the contribution! LGTM!

**Pull request merged**